app/models/item.rb

```diff
@@ -0,0 +1,4 @@
+class Item < ApplicationRecord
+  validates :name, presence: true, length: { maximum: 30 }
+  validates :description, length: { maximum: 200 }
```

Comment:
it's because of our views - we wanted to make sure it would look good, and optionally change limit later